tools/admin-edit/index.html

```diff
@@ -31,12 +31,12 @@ <h1>Admin Edit</h1>
         <section class="form-field config-field">
           <div class="form-field org-field">
             <label for="org">Organization</label>
-            <input name="org" id="org" list="org-list" required />
+            <input name="org" id="org" list="org-list" autocomplete="off" required />
```

Comment:
browser was autocompleting sites not in the org, hence setting `autocomplete="off"` to prevent that confusion

Reply:
good catch. 🎉